security/deployment.adoc

```diff
@@ -48,6 +48,135 @@ $ oc set triggers dc/frontend \
 ** xref:../dev_guide/deployments/basic_deployment_operations.adoc#triggers[Setting Deployment Triggers]
 ** xref:../dev_guide/application_lifecycle/promoting_applications.adoc#dev-guide-promoting-applications[Application Life Cycle Management -> Promoting Applications Across Environments]
 
+[[security-deployment-from-where-images-deployed]]
+== Controlling What Image Sources Can Be Deployed
+
+It is important that the intended images are actually being deployed, that
+they are from trusted sources and they have not been altered. Cryptographic
+signing provides this assurance. {product-title} enables cluster administrators
+to apply security policy that is broad or narrow, reflecting deployment
+environment and security requirements. Two parameters define this policy:
+
+* registry, with optional project namespace
+* trust: accept, reject or require public key(s)
+
+With these policy parameters, registries or parts of registries, even individual
+images, may be whitelisted (accept), blacklisted (reject), or define a trust
+relationship using trusted public key(s) to ensure the source is cryptographically
+verified. The policy rules apply to nodes. Policy may be applied uniformly
+across all nodes or targeted for different node workloads (e.g. build, zone or
+environment).
+
+.Example Policy File
+====
+----
+{
+    "default": [{"type": "reject"}],
+    "transports": {
+        "docker": {
+            "registry.access.redhat.com": [
+                {
+                    "type": "signedBy",
+                    "keyType": "GPGKeys",
+                    "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+                }
+            ]
+        },
+        "atomic": {
+            "172.30.1.1:5000/openshift": [
+                {
+                    "type": "signedBy",
+                    "keyType": "GPGKeys",
+                    "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+                }
+            ],
+            "172.30.1.1:5000/production": [
+                {
+                    "type": "signedBy",
+                    "keyType": "GPGKeys",
+                    "keyPath": "/etc/pki/example.com/pubkey"
+                }
+            ],
+            "172.30.1.1:5000": [{"type": "insecureAcceptAnything"}]
+        }
+    }
+}
+----
+====
+
+This policy enforces the following rules:
+
+.  Require images from the Red Hat registry are signed by the Red Hat public key
+.  Require images from the internal registry in the 'openshift' namespace are signed by the Red Hat public key
+.  Require images from the internal registry in the 'production' namespace are signed by example.com's public key
```

Comment:
Trying to call the "internal registry" (or any other permutation) just the "OpenShift Container Registry".